pkg/block/index.go

```diff
@@ -353,7 +370,14 @@ func GatherIndexHealthStats(ctx context.Context, logger log.Logger, fn string, m
 				stats.DuplicatedChunks++
 				continue
 			}
+			// Refer to Prometheus #12874
+			if c.MinTime == c0.MaxTime {
+				stats.DuplicatedSample++
+				level.Warn(logger).Log("msg", "found duplicate sample", "labels", lset, "index", i, "chuck", c, "chuck_last_one", c0)
```

Comment:
```suggestion
				level.Warn(logger).Log("msg", "found duplicate sample", "labels", lset, "index", i, "chunck", c, "chubck_last_one", c0)
```

Also `c` and `c0` are not printed by the loger, I get `unsuported value type`.

Reply:
Maybe we can just remove the two fields or simply print their chunk ref